setup.py

```diff
@@ -42,6 +42,7 @@
         "ipython>=6.2.1,<7.0.0",
         "plyvel==1.0.5",
         "web3==4.4.1",
+        "websockets==5.0.1",
```

Comment:
IMO better specify `"websockets>=5.0.0,<6.0.0"` - unless there's a specific reason for `5.0.1`?..

`websockets` is already pulled in as a nested dependency, so this'll make it easier to avoid breaks.

Reply:
Hrm, as long as this is within the range that `web3.py` uses, specific version pinning is actually appropriate.  We consider trinity to be an *Application* and thus, we've chosen to pin to specific versions to avoid odd errors from changes to other libraries which would otherwise cause breakage.

I'm open to discussion on the validity of this tactic since one *could* make the argument that pinning between major versions should be sufficient.  We've also talked about pinning within `minor` versions as a middle ground.

Reply:
FYI, python 3.7 support for `websockets` was added in `6.0.0`.

Reply:
Looks like we'll need to bump `web3.py` to something `>4.5.0` (but still pin at exact version)

Reply:
If any test failures show up from that version bump, lets address them separate from this PR.

Reply:
Will leave this for now as is (pinned to 5.0.1). Can be bumped later along with web3.py.

Bumping web3.py seems out of scope of this issue, but I can do it if needed.